README.md

```diff
@@ -53,7 +53,7 @@ put your business on the path to transformative growth. Disrupt your industry wi
 
 ### Cloud Database
 
-- [MongoDB Startup Accelerator](https://www.mongodb.com/startup-accelerator) - The MongoDB Startup Accelerator offers access to MongoDB Atlas, the fully-managed global cloud database, and support to help startups build and scale their businesses.
+- [MongoDB for Startups](https://www.mongodb.com/) - The MongoDB for Startups program offers access to MongoDB Atlas, the fully-managed global cloud database, and support to help startups build and scale their businesses.
```

Comment:
Can you please update the URL to point to the [MongoDB for Startups](https://www.mongodb.com/startups) page?